### PR TITLE
Updates to readme, incl Miro items

### PR DESCRIPTION
**Pull request by bethz**
add ref to pypi 0.2.0 for current release, mention current work, at top:point to info for current users and add bz as maintainer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +27,7 @@ DP refers to Demographic Parity, EO to Equalized Odds, and BGL to Bounded Group
 
 To request additional algorithms or fairness definitions, please open a new issue.
 
+# <a name="existing"></a>
```
Comment (riedgar-ms) [inline README.md]: Small thing.... can you do
`<a name="existing" />`

Comment (bethz) [reply README.md]: why is that preferred over a properly terminated </a>?

Comment (riedgar-ms) [reply README.md]: In theory, that should be properly terminated, as a self-closing tag. However, I don't have very strong opinions.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,19 @@
 
 # Fairness in machine learning - Mitigation algorithms
 
-A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning including:
+A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning.
+
+## Current Release
+
+*  The current stable release is available at [FairLearn v0.2.0](https://github.com/microsoft/fairlearn/tree/v0.2.0 ).
+
```
Comment (MiroDudik) [inline README.md]: Remove extra space after * and extra space after v0.2.0 (the link doesn't work)

File: README.md [hunk 3]
```diff
@@ -2,7 +2,19 @@
...
+* This repo contains our latest development work and is frequently updated.
+
+* Current users can reference [Existing users: How to onboard to fairlearn v0.3+](#existing).
+
```
Comment (MiroDudik) [inline README.md]: Change this bullet to something like "Our latest work differs substantively from version 0.2.0, please see [Existing...]"

File: README.md (hunk 2 above)
Comment (bethz) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -2,7 +2,19 @@
...
 # Fairness in machine learning - Mitigation algorithms
 
-A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning including:
+A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning.
+
+## Current Release
+
+* The current stable release is available at [FairLearn v0.2.0](https://github.com/microsoft/fairlearn/tree/v0.2.0).
+
+* Our latest work differs substantively from version 0.2.0, please investigate this repo for further information.
```
Comment (romanlutz) [inline README.md]: What do you mean by investigate here? I'm a little confused.

Comment (bethz) [reply README.md]: changed to please visit....

File: README.md (hunk 3 above)
Comment (bethz) [reply README.md]: done

**Pull request merged**